src/Bonami/HoneyBunny/ConsumerDaemon.php

```diff
@@ -44,6 +48,8 @@ public function __construct(
 		$prefetchSize,
 		$prefetchCount,
 		IMessageConsumer $messageConsumer,
+		$exchangeName = null,
+		$routingKey = null,
```

Comment:
Messaging v rabbitu funguje vzdy takhle
`producer -> exchange -> (binding) -> queue -> consumers`
tzn. vzdycky publikujes zpravu do exchange. Analogie muze byt exchange = postovni urad, queue = schranka u baraku.
I v nejvic simple pripade (tak jak jsme vyhradne pouzivali rabbita my) publikujes do exchange, ale trik je v tom, ze rabbit sam vytvori default `direct` exchange (jmeno exchange je prazdny string), ke ktere nabinduje kazdou vytvorenou frontu tak, ze routing key = nazev fronty.

Pokud tedy napr. posilas zpravu do fronty `device_hit`, ve skutecnosti posilas zpravu do default direct exchange s routing klicem `device_hit`.

No a ted uz je asi jasne, proc to cele pisu - mame tu mozna vic parametru nez je nutne, protoze `queueName` je to same co `routingKey`...

Reply:
Myslím, že jsme pořešili osobně :-)